src/reducers/browse.js

```diff
@@ -25,6 +25,20 @@ function browse(state = initialBrowseState, action) {
     case actionTypes.GET_SCHOOLS_FAILURE:
       return state.set('isFetching', false)
         .set('schools', initialBrowseState.get('schools'));
+    case actionTypes.FETCH_SUBJECTS_REQUEST:
```

Comment:
Let's put a space above this line to separate the subject fetch handling logic.

Reply:
Will do!

Reply:
Had to put `// eslint-disable-next-line no-trailing-spaces` in for this. Not sure if that's the right way to approach it

Reply:
That's for whitespace after a line, not for empty lines. If you're getting that linting error it means you have trailing whitespace after a line of code somewhere.

Reply:
We still need to remove the `eslint-disable`